releasenotes/notes/1.0/prepare-1.0-e13b9b8273055775.yaml

```
@@ -0,0 +1,69 @@
+---
+prelude: |
+  We are very please to release the Qiskit 1.0.0 release. This release is the
```

I know you saw it, but couldn't resist commenting (also, the frequency of the word"release" in this line is maybe a bit too high):

```suggestion
  We are very pleased to release Qiskit 1.0.0. This release is the
```